[Homework #19] React-api by Abraham Trenyan

AbrahamTrenyan opened the pull request:
Hi Vitali! I've been struggling mostly with the percentage functionality, but finally managed to make it work.

VitalikMelnyk commented:
It necessary some fixes:
1. Format all files with Prettier.
2. Fix the design. More info on the screenshot. It is about one text is overlapped on another, and when a lot of numbers, we have an overflow and don't see all numbers and operations.
<img width="470" alt="Screenshot 2022-09-30 at 12 24 23" src="https://user-images.githubusercontent.com/46888925/193239842-af2a3b4c-b174-4cdd-b3bc-f61be230945d.png">

VitalikMelnyk commented on homeworks/abraham-trenyan/19-react-api/src/App.js:
```diff
@@ -0,0 +1,116 @@
+import Button from './Components/Button/Button';
+function App() {
+  const [buttons] = useState([
+    'C',
```
Export to constants in `utils` or `helpers` folder

VitalikMelnyk commented on homeworks/abraham-trenyan/19-react-api/src/App.js:
```diff
@@ -0,0 +1,116 @@
+    } else {
+      // If the percentage sign is found after another sign, it would behave like this: takes the number before the percentage sign and divides it by 100, then multiplies it by the number before the other sign, because it will be a percentage of this other number.
+      let numberToDivideByHundred = percentOperation.slice(lastSignIndex + 1, percentIndex).join(''); // Takes the part of the operation since the last sign to the percentage sign, that is the number affected by the sign that we should divide by 100.
+      let multiplier = Function('return ' + percentOperation.slice(0, lastSignIndex).join(''))();
```
![Screenshot 2022-09-30 at 12 34 57](https://user-images.githubusercontent.com/46888925/193241163-6035f092-5dbc-4a32-b1a3-bd336640af4c.png)
In the future should be cautious with this.

VitalikMelnyk commented on homeworks/abraham-trenyan/19-react-api/src/App.js:
```diff
@@ -0,0 +1,116 @@
+            key={i}
+            value={btn}
+            onClick={
+              btn === 'C'
```
Refactor this because it isn't readable, as for me.

AbrahamTrenyan commented:
I set a limit for numbers to avoid overflow, fixed mobile design, format files with prettier, moved buttons to utils folder, renamed components folder  and refactored onclick function. Thanks for the comments!
